mixer/cmd/build.go

```diff
@@ -266,7 +260,7 @@ func init() {
 	externalDeps[buildChrootsCmd] = []string{
 		"m4",
 		"rpm",
-		"yum",
+		"dnf",
```

Comment:
Yum is still a dependency if you try to run without --new-chroots (not a mixer dependency, but a dependency still).
I guess this is ok since the old one will not work on new systems anyway, but it's a small inconsistency for the time being.

Reply:
It seems the old one is moving to only using dnf too, so I think we are good here.